testing/snapshot.ts

```diff
@@ -29,7 +29,7 @@ function serialize(actual: unknown): string {
     compact: false,
     iterableLimit: Infinity,
     strAbbreviateSize: Infinity,
-  });
+  }).replace(/\\n/g, "\n");
```

Comment:
With this change, the expression `"..."` isn't canonical string literal anymore if it contains `\n`.

I think it's ok and works in most cases.

But does anyone have any concern about this transformation?

Reply:
Yeah this is also how it is in Jest but that doesn't necessarily mean it's how it should be done here. The alternative I considered was:

```ts
await assertSnapshot("Hello\nWorld!");

export[`snapshot`] = `
\`Hello
World!\`
`;
```

But I felt that this has two downsides:

1. The snapshot is less readable due to the presence of escape characters
2. It would require significant changes to `Deno.inspect` and I was concerned about cluttering the API with options that have no use case (at least not that I can think of!) outside of snapshot testing

Reply:
> With this change, the expression `"..."` isn't canonical string literal anymore if it contains `\n`.
> 
> I think it's ok and works in most cases.
> 
> But does anyone have any concern about this transformation?

@kt3k Are we still waiting for feedback on this?

Reply:
Let's land and try to see the community feedback. Thanks for your contribution!